plugins/meta/bandwidth/main.go

```diff
@@ -112,13 +114,11 @@ func validateRateAndBurst(rate int, burst int) error {
 }
 
 func getIfbDeviceName(networkName string, containerId string) (string, error) {
-	hash := sha1.New()
-	_, err := hash.Write([]byte(networkName + containerId))
+	hash, err := hash.New(networkName+containerId, maxIfbDeviceLength-3)
```

Comment:
Is `-3` correct? I see that you append "bw" to the return value which has only a length of `2`

Reply:
Yeah, it'd much rather have 
```golang
const ifbDeviceNamePrefix = "bw"
```
and
```golang
hash.New(..., maxIfbDeviceLength-len(ifbDeviceNamePrefix))
```